src/index.d.ts

```diff
@@ -9,7 +9,7 @@ import * as React from "react";
  * @param promise Input promise to be tracked.
  * @returns It returns the same promise as input.
  */
-export function trackPromise(promise: Promise<any>): Promise<any>;
+export function trackPromise<T>(promise: Promise<T>, area: string): Promise<T>;
```

Comment:
Area is a non mandatory field, should it be?

```javascript
export function trackPromise<T>(promise: Promise<T>, area?: string): Promise<T>;
```

Reply:
Ouch, yes you are right, I forgot to make it optional.